spec/ruby/core/enumerable/grep_v_spec.rb

```diff
@@ -0,0 +1,34 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+
+describe "Enumerable#grep_v" do
+  it "returns an array of all elements not === pattern when called without a block" do
+    pattern_cls = Class.new do
+      def ===(obj)
+        obj == '2'
+      end
+    end
+    enum = EnumerableSpecs::Numerous.new('2', 'a', 'nil', '3', false)
+
+    enum.grep_v(pattern_cls.new).should == ['a', 'nil', '3', false]
+  end
+
+  it "returns an array of all elements not === pattern passed through the block" do
```

Comment:
This description is a little bit confusing. I think something like `of all elements not equal to the pattern ....` would be better (at least if that's what the description is meant to state).